backend/typescript/middlewares/validators/authValidators.ts

```diff
@@ -45,10 +45,12 @@ export const registerRequestValidator = async (
   }
 
   if (req.body.role === Role.VOLUNTEER) {
+    if (!Object.values(Status).includes(req.body.status)) {
```

Comment:
i think we should check if status was included in the body or not, so smth `if (req.body.status && !Object.values(Status).includes(req.body.status))` because the default is already set as Pending and frontend will not pass anything for status